.github/workflows/check_docs.yml

```diff
@@ -0,0 +1,35 @@
+name: 'check docs'
```

Comment:
Same request as others to remove this file

Reply:
removed